cluster/charts/rook-ceph/templates/role.yaml

```diff
@@ -37,18 +37,6 @@ rules:
   - create
   - update
   - delete
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - servicemonitors
-  - prometheusrules
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
-  - delete
```

Comment:
This is also vestigial and covered by monitoring-specific resources that now exist.